app/os.go

```diff
@@ -43,6 +45,10 @@ type Config struct {
 	CustomRenderer bool
 	// Decorated reports whether window decorations are provided automatically.
 	Decorated bool
+
+	// id used for Wayland and X11 app_id.
+	id string
```

Comment:
Why not read `ID` directly from `os_wayland.go`, avoiding the overhead of an option?